engine/gameobject/src/dmsdk/gameobject/component.h

```diff
@@ -678,7 +681,9 @@ namespace dmGameObject
         uint8_t DM_ALIGNED(16) DM_COMPONENT_PASTE_SYMREG2(symbol, __LINE__)[dmGameObject::s_ComponentTypeDescBufferSize]; \
         DM_REGISTER_COMPONENT_TYPE(symbol, DM_COMPONENT_PASTE_SYMREG2(symbol, __LINE__), name, type_create_fn);
 
-
+#ifndef MIN
+   #define MIN(x, y) ((x) < (y) ? x : y)
+#endif
```

Comment:
I don't wanna include new .h file in each component. This simple macros is for that. But maybe I'm wrong here

Reply:
1) Why is this needed? Why isn't "dmMath::Min" good enough?
2) That name is way too generic. It's very likely to clash with defines from other people's code.

Reply:
`dmMath::Min` is perfect, thank you!